atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/DbKvsPostgresInvalidationRunnerTest.java

```diff
@@ -31,28 +31,46 @@
 import java.time.Duration;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicInteger;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 @ExtendWith(DbKvsPostgresExtension.class)
 public class DbKvsPostgresInvalidationRunnerTest {
+    private static final AtomicInteger prefixCounter = new AtomicInteger(0);
+
     @RegisterExtension
     public static final TestResourceManager TRM = new TestResourceManager(DbKvsPostgresExtension::createKvs);
 
     private final ConnectionManagerAwareDbKvs kvs = (ConnectionManagerAwareDbKvs) TRM.getDefaultKvs();
-    private final TimestampBoundStore store = getStore();
-    private final InvalidationRunner invalidationRunner = new InvalidationRunner(
-            kvs.getConnectionManager(),
-            AtlasDbConstants.TIMESTAMP_TABLE,
-            DbKvsPostgresExtension.getKvsConfig().ddl().tablePrefix());
+    private TimestampBoundStore store;
+    private InvalidationRunner invalidationRunner;
     private static final long TIMESTAMP_1 = 12000;
 
     @BeforeEach
     public void setUp() {
         kvs.dropTable(AtlasDbConstants.TIMESTAMP_TABLE);
+        String prefix = nextUniquePrefix(prefixCounter);
+        invalidationRunner =
+                new InvalidationRunner(kvs.getConnectionManager(), AtlasDbConstants.TIMESTAMP_TABLE, prefix);
         invalidationRunner.createTableIfDoesNotExist();
+        store = getStoreWithPrefix(prefix);
+    }
+
+    // 0 -> "a", 1 -> "b", ..., 25 -> "z", 26 -> "aa", 27 -> "ab", ...
+    private static String nextUniquePrefix(AtomicInteger counter) {
+        int nextValue = counter.getAndIncrement();
+
+        StringBuilder result = new StringBuilder();
+        while (nextValue >= 0) {
+            int remainder = nextValue % 26;
+            result.insert(0, (char) ('a' + remainder));
+            nextValue = (nextValue / 26) - 1;
+        }
```

Comment:
Ah, fair point. For whatever reason the first solution in my mind to the "you can't start the name with a number" was to convert it to a sequence of non-numeric characters rather than simply adding a non-numeric single char prefix and then add the number 🤦